packages/actor-dereference-file/lib/ActorDereferenceFile.ts

```diff
@@ -22,13 +22,16 @@ export class ActorDereferenceFile extends ActorDereference {
   }
 
   public async run({ url }: IActionDereference): Promise<IActorDereferenceOutput> {
+    const URIRegex = /\w[\w+.-]*:.*/u;
```

Comment:
Why are these characters needed in the regex?: `+.-`
I would guess they can be removed?

Reply:
Also, I suggest moving this regex as constant readonly field on the class, so that we only have to create it at load-time.

Reply:
> Why are these characters needed in the regex?: +.-

I haven't seen any  URI schemes with `+.-` in them, so it's been to my surprise to learn that they make for a valid scheme, according to [this wiki page](https://en.wikipedia.org/wiki/URL)
![image](https://user-images.githubusercontent.com/13056889/211488952-6d484dde-3bbc-4f18-825f-39a3bb42d33f.png)

Do you think we're better off dropping them?

Reply:
I see, good catch! Let's keep them there then :-)